src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/unit/service/cas1/Cas1BookingEmailServiceTest.kt

```diff
@@ -392,6 +397,9 @@ class Cas1BookingEmailServiceTest {
         arrivalDate = LocalDate.of(2023, 2, 1),
         departureDate = LocalDate.of(2023, 2, 14),
         caseManagerNotApplicant = true,
+        cruManagementArea = Cas1CruManagementAreaEntityFactory()
+          .withEmailAddress(AP_AREA_EMAIL)
```

Comment:
could we please rename this constant e.g. `CRU_MANAGEMENT_AREA_EMAIL`

presumably this is being set on the ap area somewhere in the test? We should stop doing that to prove we're using the CRU Management Area Email

Reply:
Renamed this.